content/cti/index.md

```diff
@@ -24,25 +25,29 @@ Download the Community Training Image (CTI)
 </center>
 
 ## Getting Started with the CTI
-1. [Download the CTI VM](https://downloads.gcentral.org/vm/20231021_OpenSUSE_CTIBase.ova) (~ 8 GB download). 
-2. Download the setup and getting started instructions in either [PDF](https://downloads.gcentral.org/Tutor%20Instructions%20Course%201%20-%20v0.1.0.pdf) or [ODT](https://downloads.gcentral.org/Tutor%20Instructions%20Course%201%20-%20v0.1.0.odt) format. 
+
+1. [Download the CTI VM](https://downloads.gcentral.org/vm/20240327_Lubuntu_CTIBase.ova) (~ 4.8 GB download).
+2. Download the setup and getting started instructions in either [PDF](https://downloads.gcentral.org/Tutor%20Instructions%20Course%201%20-%20v0.1.0.pdf) or [ODT](https://downloads.gcentral.org/Tutor%20Instructions%20Course%201%20-%20v0.1.0.odt) format.
```

Comment:
No change in documentation version eventhough there is a new image with a different linux distro?

Reply:
So far no change, although Steve did indicate he'd be updating the documentation. 

The information in the current documentation did allow me to log in, etc.